### DISCUSSION TITLE
Add severity @tags within pytest, pytest-bdd with gherkin (using feature files)

### DISCUSSION THREAD

**Discussion by MihaiDorelRus** [allure-framework/allure2#1570] [Questions & Support] [answered]
Hello,

I am using allure reporting in a Python project (with Pytest, Pytest-bdd & gherkin steps - feature files). I am trying to add @severity tags above the Scenario, but it's not taken into consideration. Is there another way of adding severity for my tests?

Thanks

**Comment by storenth** [comment, marked as answer]
I create the issue based on TestOps unexpected behavior 
https://github.com/allure-framework/allure-python/issues/659
